docs/ops/generation/Range_4.md

```diff
@@ -0,0 +1,82 @@
+## Range<a name="Range"></a>
+
+**Versioned name**: *Range-4*
+
+**Category**: Generation
+
+**Short description**: *Range* operation generates a sequence of numbers according input values [start, stop) with a step.
+
+**Attributes**:
+
+* *output_type*
+
+  * **Description**: the output tensor type
+  * **Range of values**: u1, u8, u16, u32, u64, i8, i16, i32, i64, f16, f32, boolean, bf16
+  * **Type**: string
+  * **Default value**: None
+  * **Required**: *Yes*
+
+**Inputs**:
+
+* **1**: "start" - A scalar of type T. **Required.**
```

Comment:
T -> *T*
Also it seems *T* should be equal to `output_type`, otherwise, how to handle a case when `start`=1.5 and `output_type`=i32? Please make a note about it.

Reply:
I am not sure what would be best. Input to Range-4 might be int8, but output int32 for example, and it makes sense to me, we need to discuss that issue.